lemur/certificates/cli.py

```diff
@@ -761,7 +762,10 @@ def deactivate_entrust_certificates():
 
     certificates = get_all_valid_certs(['entrust-issuer'])
     entrust_plugin = plugins.get('entrust-issuer')
-    for cert in certificates:
+    for index, cert in enumerate(certificates, start=1):
```

Comment:
we can have start=0, so that it will deactivate 10 certs in each batch. Else it will be 9 in first batch and 10 for the rest.

Reply:
good catch!